doc/dev/debugging.rst

```diff
@@ -196,6 +196,27 @@ the following command.
 
     circuit.get_output_of("BufferizationPass")
```

Comment:
@dime10 should we update this (as well as the `Compilation Steps` section to use `print_compilation_stage` instead? That debugging function is not linked to from this page.

Reply:
@dime10 is this something you would be able to quickly do? I'm not sure exactly how the new functionality works to confidently update it

Reply:
I guess this has been tackled in #714.

@josh146 does this mean the docstring in `print_compilation_stage` is inadequate?